src/homeWork13/SortAlgorithms.java

```diff
@@ -0,0 +1,30 @@
+package homeWork13;
+
+public class SortAlgorithms {
+
+    public static int[] insertionSort(int[] array) {
+        for (int j = 1; j < array.length; j++) {
+            int key = array[j];
+            int i = j - 1;
+            while ((i >= 0) && (array[i] > key)) {
```

Comment:
You can simplify this `((i >= 0) && (array[i] > key))` as `(i >= 0 && array[i] > key)`